### DISCUSSION TITLE
Improvement suggestion - listening directly from tooltip.

### DISCUSSION THREAD

**Discussion by bkoltoniak** [sienori/simple-translate#317] [Ideas]
When I am using simple translate, I often use the listening feature. This helps me in learning how correctly pronounce unfamiliar words. 

Typically, am using the tooltip on selected text to translate words or text. This is quite comfortable. Unfortunately form tooltip level is impossible to listen the word pronounce. It would be super handy if listening feature will be available directly from tooltip.

Please consider adding such possibility :) Simple translate is robust and nice plugin. Keep it up 👍

**Comment by abdul-basit-bboxx** [comment]
Hey @bkoltoniak, are you using this feature in your local machine ?
If yes, please share you PR I'd like to use.

**Comment by chwdWHO** [comment]
+1

**Comment by nwidiastomo** [comment]
adding a way to read in latin alphabet (for example, russian cyrillic or japanese kanji) might be a nice touch too

**Comment by staufenberg** [comment]
i'am missing this too. 
That couldn't be that hard: https://www.codeproject.com/Articles/5299196/Text-to-Speech-with-Google-Translate
